### PR TITLE
Updates to the web UI

### LINKED ISSUE #7217: Stack trace box size is unbounded
This causes the labels like "Error Type" to wrap lines. We should also add a copy to clipboard button.

### LINKED ISSUE #7240: UI query detail page: long session strings overflow to next column
A long user provided string such as User, Principal, Source or Resource Group will overflow to the Data Source section.

### PR DESCRIPTION
**Pull request by raghavsethi**
Fixes #7217 
Fixes #7240

### CODE REVIEW

File: presto-main/src/main/resources/webapp/assets/utils.js [hunk 1]
```diff
@@ -90,16 +90,22 @@ function getStageStateColor(state)
 function getHumanReadableState(query)
 {
     if (query.state == "RUNNING") {
+        let title = "RUNNING";
+
         if (query.scheduled && query.queryStats.totalDrivers > 0 && query.queryStats.runningDrivers >= 0) {
-            return "RUNNING";
-        }
+            if (query.queryStats.fullyBlocked) {
+                if (!query.queryStats.blockedReasons || query.queryStats.blockedReasons.length) {
```
Comment (electrum) [inline presto-main/src/main/resources/webapp/assets/utils.js]: Should this be `length > 0`? Or is this idiomatic in JS?

File: presto-main/src/main/java/com/facebook/presto/execution/StageStateMachine.java [hunk 2]
```diff
@@ -274,8 +275,11 @@ public StageInfo getStageInfo(Supplier<Iterable<TaskInfo>> taskInfosSupplier, Su
             outputDataSize += taskStats.getOutputDataSize().toBytes();
             outputPositions += taskStats.getOutputPositions();
 
-            taskStats.getPipelines().forEach(pipeline -> pipeline.getOperatorSummaries()
-                            .forEach(operatorStats -> operatorToStats.compute(operatorStats.getOperatorId(), (k, v) -> v == null ? operatorStats : v.add(operatorStats))));
+            for (PipelineStats pipeline : taskStats.getPipelines()) {
+                for (OperatorStats operatorStats : pipeline.getOperatorSummaries()) {
+                    operatorToStats.compute(pipeline.getPipelineId() + "." + operatorStats.getOperatorId(), (k, v) -> v == null ? operatorStats : v.add(operatorStats));
```
Comment (electrum) [inline presto-main/src/main/java/com/facebook/presto/execution/StageStateMachine.java]: Introduce a variable:
```java
String id = pipeline.getPipelineId() + "." + operatorStats.getOperatorId();
operatorToStats.compute(id, (k, v) -> v == null ? operatorStats : v.add(operatorStats));

**Comment by raghavsethi** [comment]
@erichwang Need a review for the 'Add pipeline ID to PipelineContext' commit.

File: presto-main/src/main/java/com/facebook/presto/operator/PipelineStats.java [hunk 3]
```diff
@@ -76,6 +78,8 @@
 
     @JsonCreator
     public PipelineStats(
+            @JsonProperty("pipelineId") int pipelineId,
+
```
Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/operator/PipelineStats.java]: extra blank line

File: presto-main/src/main/java/com/facebook/presto/operator/PipelineStats.java [hunk 4]
```diff
@@ -115,6 +119,8 @@ public PipelineStats(
             @JsonProperty("operatorSummaries") List<OperatorStats> operatorSummaries,
             @JsonProperty("drivers") List<DriverStats> drivers)
     {
+        this.pipelineId = pipelineId;
+
```
Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/operator/PipelineStats.java]: extra blank line

File: presto-main/src/main/java/com/facebook/presto/operator/index/IndexLoader.java [hunk 5]
```diff
@@ -60,6 +60,7 @@
 public class IndexLoader
 {
     private static final ConnectorId INDEX_CONNECTOR_ID = new ConnectorId("$index");
+
```
Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/operator/index/IndexLoader.java]: extra blank line

**Pull request closed without merging**